index.js

```diff
@@ -1,98 +1,109 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-unused-vars */
```

Comment:
- I'll advise you to remove all disabled linters and try to fix the issues. Remember that linters help you to ensure that you write code that is clean and easy to maintain Please fix this 🙏

Reply:
hello, for the links switching to the right page section, this is the project requirement for the third section of this project. this is currently the second part which is refactor using classes. Hence for this project it's not part of the requirement

Reply:
This we are required to implement in the third project.

Reply:
Okay @george827, I noticed it was implemented in the previous milestone(PR), so I thought you introduced a bug in the code.

Reply:
Hi @mckent05  sorry for the confusion 🙏. Thank you for the hint you gave.